packages/frontend/src/utils/sortUtils.ts

```diff
@@ -94,6 +95,34 @@ export const getSortLabel = (
         }
     }
 
+    if (isCustomSqlDimension(item)) {
```

Comment:
Show sort options based on the return type the user set for the custom dimension.